features/proposed/add-blocklist/README.md

```diff
@@ -0,0 +1,64 @@
+# Integrate Botnet Controller List (BCL)
+
+## Problem Summary
+
+Currently, Firefox Accounts uses [customs server](https://github.com/mozilla/fxa-customs-server) to detect fraud and abuse.
+This server is capable of blocking and rate limiting requests from a variety of sources.
+The primary problem is that the custom server does not take advantage of known blocklists when evaluating whether or not a request should be processed.
+
+There are multiple BCLs, each with varying rates of success and false positives. This feature does not specify a specific blocklist that will be used but rather the format needed by the list, how to integrate and how to handle false postives. Some of the major blocklist sites include [Spamhaus BCL](https://www.spamhaus.org/bcl/) and [FireHOL](http://iplists.firehol.org/).
```

Comment:
Here also, "BCL" refers to a specific list, let's use the generic term "blocklist".